From 8c764ab29bc1d21794d9052bf89995a79850f173 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 10 Jan 2020 17:24:53 -0800
Subject: [PATCH 0800/1852] dmaengine: xilinx: frmbuf: Add support for 8 ppc

commit 243c88799703bd440be38faee01666200bed678a from
https://github.com/Xilinx/linux-xlnx.git

Add support for 8 pixels per clock by allowing it as a valid dt
parameter value.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/dma/xilinx/xilinx_frmbuf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/xilinx/xilinx_frmbuf.c b/drivers/dma/xilinx/xilinx_frmbuf.c
index f784eda90156..c5ba71991f2b 100644
--- a/drivers/dma/xilinx/xilinx_frmbuf.c
+++ b/drivers/dma/xilinx/xilinx_frmbuf.c
@@ -1559,7 +1559,7 @@ static int xilinx_frmbuf_probe(struct platform_device *pdev)
 	/* Initialize the DMA engine */
 	if (xdev->cfg->flags & XILINX_PPC_PROP) {
 		err = of_property_read_u32(node, "xlnx,pixels-per-clock", &ppc);
-		if (err || (ppc != 1 && ppc != 2 && ppc != 4)) {
+		if (err || (ppc != 1 && ppc != 2 && ppc != 4 && ppc != 8)) {
 			dev_err(&pdev->dev, "missing or invalid pixels per clock dts prop\n");
 			return err;
 		}
-- 
2.31.1

